app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1482,10 +1482,6 @@ public void handleResult(@NonNull final StreamInfo info) {
             displayUploaderAsSubChannel(info);
         }
 
-        final Drawable buddyDrawable =
-                AppCompatResources.getDrawable(activity, R.drawable.placeholder_person);
-        binding.detailSubChannelThumbnailView.setImageDrawable(buddyDrawable);
-        binding.detailUploaderThumbnailView.setImageDrawable(buddyDrawable);
```

Comment:
Thank you for the research! You correctly identified these problematic lines to remove. However, I don't think we need to add the streamAvatar to the database. Could you explain why you did so? Maybe I'm missing something.
In the meantime I opened #10581 which supersedes this one.